.github/workflows/_integration_tests.yml

```diff
@@ -118,12 +124,16 @@ jobs:
         run: docker compose run elixir /bin/sh -c 'cd apps/domain && mix ecto.seed'
       - name: Start docker compose in the background
         run: |
+          echo "FIREZONE_DNS_CONTROL=${{ matrix.dns-control }}" >> .env
+          cat .env
```

Comment:
Is this a debug line?

Reply:
Yeah, I'll get rid of it since the method for controlling env vars ended up being simple

Could I do something in another PR to reduce the "Downloading a billion crates... compiling a billion crates..." noise that Cargo always puts out? It makes it hard to scroll through the Github Actions logs, especially for something like `cargo test` where there is output I really need to see, at the end of 1,000 lines of basically progress bars.